pootle/core/models/snapshot.py

```diff
@@ -90,8 +92,10 @@ def save(self, ctx_data):
         except OSError:
             pass
 
-        with open(self.filepath, 'w') as outfile:
-            outfile.write(ctx_data)
+        # use newline='' not to convert \n to OS-specific newlines;
+        # this is to be able to generate snapshots on any platform
+        with io.open(self.filepath, 'w', newline='') as outfile:
+            outfile.write(unicode(ctx_data))
```

Comment:
I was researching this topic yesterday and there was a notice that the way `wb` works will change in Python 3, and that the code above is a more 'modern' way of doing this. Can you comment on that?